unit_test/test_g2c_degrib2c_grib2.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+#            Test G2C Library 
+#    This script uses to test G2C library with following routines:
+#    - seekgb.c ; fseek; fread; g2_info; g2_getfld; g2_free;
+#      g2_unpack1; g2_unpack2; g2_unpack3, g2_unpack4, g2_unpack5;
+#      g2_unpack6; g2_unpack7
+
+set +x
+
+echo " "
+echo "###################################################"
+echo " Testing G2C Library "
+echo "###################################################"
+echo " "
+
+export dir=` pwd `
+
+. $MODULESHOME/init/bash
+mac=$(hostname | cut -c1-1)
+mac2=$(hostname | cut -c1-2)
+
+if [ $mac = v -o $mac = m  ] ; then            # For Dell (i.e. Mars and Venus)
+   echo ""
+   export machine=wcoss_dell_p3
+   echo ""
+elif [ $mac = l -o $mac = s ] ; then             #    wcoss_c (i.e. luna and surge)
+   echo ""
+   export machine=cray-intel
+   echo ""
+fi
+
+echo " "
+echo " You are on WCOSS:  $machine "
+echo " "
+
+sleep 2
+
+export cyc=00
+export cycle=t${cyc}z
+export exec=$dir
+
+#
+#  Setup working directories
+#
+# If you want to use temporary directories,
+# you can change variable dir to temporary
+#
+export data=/gpfs/dell2/stmp/${LOGNAME}/data
```

Comment:
This is searching for a set of data files here on WCOSS machines? We need those files to be tarred up and uploaded to our FTP site. The test code will retrieve them from there. GitHub CI does not have access to WCOSS directories.